<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TitleWidget" name="titleWidget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="verticalWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>用户登录</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="userMsgWidget" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>用户名：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="pwdLogin">
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="userLogin"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="utilWidget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QCheckBox" name="pwdCheckBoxLogin">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>记住密码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toLogupBtn">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                    <underline>true</underline>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 170, 127);</string>
                  </property>
                  <property name="text">
                   <string>没有账号，马上注册</string>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="logBtnWidget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="loginBtn">
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>22</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/resc/images/balckButton.png);
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>登录</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logupPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>105</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="verticalWidget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>用户注册</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="logupMsgWidget" native="true">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>用户名：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="userLogup"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>昵称：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="nicknameLogup"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="pwdLogup">
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>确认密码：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="confirmPwdLogup">
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>邮箱：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="emailLogup"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="utilWidget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="logupBtn">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>22</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border-image: url(:/resc/images/balckButton.png);
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>注册</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>105</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="configPage">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>122</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <spacer name="verticalSpacer_10">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="text">
             <string>服务器配置</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_11">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器地址：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="serverIp"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="serverPort"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器端口：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="spacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_12">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="configBtn">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/resc/images/balckButton.png);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_13">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>122</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header location="global">toolswt/titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userLogin</tabstop>
  <tabstop>pwdLogin</tabstop>
  <tabstop>loginBtn</tabstop>
  <tabstop>pwdCheckBoxLogin</tabstop>
  <tabstop>toLogupBtn</tabstop>
  <tabstop>userLogup</tabstop>
  <tabstop>nicknameLogup</tabstop>
  <tabstop>pwdLogup</tabstop>
  <tabstop>confirmPwdLogup</tabstop>
  <tabstop>emailLogup</tabstop>
  <tabstop>logupBtn</tabstop>
  <tabstop>serverIp</tabstop>
  <tabstop>serverPort</tabstop>
  <tabstop>configBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
